client/src/pages/PastBills.js

```diff
@@ -1,8 +1,36 @@
-import React from 'react';
-import { LineChart } from '../components';
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
+import { ToggleContainer, Table, Loader, LineChart } from '../components';
 
-const bills = [120, 60, 40, 50, 80, 25, 55, 100, 120, 20, 30, 70];
+const PastBills = () => {
+  const { t } = useTranslation();
+  const { billType } = useParams();
+  const [bills, setBills] = useState();
+  const getBills = async () => {
+    const { data } = await axios.get('/api/v1/bills/me');
+    setBills(data);
+  };
```

Comment:
```suggestion
const {bills, error} = useContext(DataContext)
```

you can get the bills from the context instead of make a request